OpenRA.Mods.Common/Widgets/Logic/Ingame/MenuButtonsChromeLogic.cs

```diff
@@ -85,8 +85,29 @@ public MenuButtonsChromeLogic(Widget widget, World world)
 			if (stats != null)
 			{
 				stats.IsDisabled = () => disableSystemButtons || world.Map.Visibility.HasFlag(MapVisibility.MissionSelector);
-				stats.OnClick = () => OpenMenuPanel(stats);
+				stats.OnClick = () => OpenMenuPanel(stats, new WidgetArgs() { { "activePanel", ObserverStatsPanel.Basic } });
 			}
+
+			var keyListener = widget.GetOrNull<LogicKeyListenerWidget>("OBSERVER_KEY_LISTENER");
+			if (keyListener != null)
+				keyListener.AddHandler(e =>
+				{
+					if (e.Event == KeyInputEvent.Down && !e.IsRepeat)
+					{
+						var ks = Game.Settings.Keys;
+						var key = new Hotkey(e.Key, e.Modifiers);
+						Hotkey[] statsKeys = { ks.StatisticsBasicKey, ks.StatisticsEconomyKey, ks.StatisticsProductionKey, ks.StatisticsCombatKey, ks.StatisticsGraphKey };
```

Comment:
Can you please use NamedHotkey instead? These hardcoded ks.* fields are about to move to yaml.

Reply:
I am not sure how to do it.
1. move Statistics<name>Key from ObserverStatsLogic to MenuButtonsChromeLogic and then pass them to ObserverStatsLogic 
2. use ObserverStatsLogic.StatsHotkeys in MenuButtonsChromeLogic. Trying to find now how to get reference to this Logic object in MenuButtonsChromeLogic
3. ?

Reply:
Hmm, good question.  I think the best approach here will be to define/accept the same Statistics*Key args in `MenuButtonsChromeLogic` that you use in `ObserverStatsLogic` -- define them in both places rather than reference or try to pass them between classes.